### PR TITLE
Added Cliengo Tag Template'

### PR DESCRIPTION
**Pull request by laziob**
I added a Tag for setting up the Cliengo Chatbot. Its not well known in US and Europe, but its being growing steady in Argentina and some other Latam markets so I thought it might be useful.

### CODE REVIEW

File: tags/Cliengo/Cliengo Chatbot/Cliengo.tpl [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+const url = 'https://s.cliengo.com/weboptimizer/'+companyId+'/'+websiteId+'.js';
+
+const setClgo = setInWindow('ldk', function() {
+  const document = copyFromWindow('document');
+  var ldk= document.createElement('script');
+  ldk.type='text/javascript'; 
+  ldk.async=true;
+  ldk.src=url;
+  var s=document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ldk, s);
+  });
```
Comment (eburley) [inline tags/Cliengo/Cliengo Chatbot/Cliengo.tpl]: This isn't called, and would try to do what injectScript is already doing. Sandboxed script doesn't allow direct document manipulation. It looks like you can omit this and the require for copyFromWindow and setInWindow.

Comment (laziob) [reply tags/Cliengo/Cliengo Chatbot/Cliengo.tpl]: Hmm I see. Itsmy first time with Sandboxed script so Im still getting around it. This is the Script I was trying to translate:
`<!-- Código de instalación Cliengo para www.palmaresonline.com.ar/ --> <script type="text/javascript">(function () { var ldk = document.createElement('script'); ldk.type = 'text/javascript'; ldk.async = true; ldk.src = 'https://s.cliengo.com/weboptimizer/companyId/websiteId.js'; var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ldk, s); })();</script>`

Are you saying that I can delete those lines directly?  (From 177 to 184)

Comment (sahava) [reply tags/Cliengo/Cliengo Chatbot/Cliengo.tpl]: @laziob could you make the change that Eric suggested and update the PR? Thanks.

**Review by laziob — COMMENTED** [review]
Simo Hi. I think I fixed it. Please let me know! Thanks!

File: tags/Cliengo/Cliengo Chatbot/Cliengo.tpl [hunk 2]
```diff
@@ -167,21 +167,11 @@ ___SANDBOXED_JS_FOR_WEB_TEMPLATE___
 
 // Introduzca aquí su código de plantilla.
 var log = require('logToConsole');
```
Comment (eburley) [inline tags/Cliengo/Cliengo Chatbot/Cliengo.tpl]: you can make this const as well.

Comment (laziob) [reply tags/Cliengo/Cliengo Chatbot/Cliengo.tpl]: Done! 😄

**Pull request merged**